crates/bevy_sprite/src/sprite.rs

```diff
@@ -1,9 +1,13 @@
-use bevy_ecs::component::Component;
+use bevy_ecs::{component::Component, reflect::ReflectComponent};
 use bevy_math::{Rect, Vec2};
-use bevy_reflect::Reflect;
+use bevy_reflect::{
+    prelude::{FromReflect, ReflectDefault},
+    Reflect,
+};
```

Comment:
Nit: I think you can also do this to clean up the import
```suggestion
use bevy_reflect::prelude::{FromReflect, Reflect, ReflectDefault};
```

Reply:
We've been avoiding manual prelude imports by convention in "built in" bevy crates.  Imo the prelude exists to be imported in full, not to be the "source of truth import path".